mne/report/report.py

```diff
@@ -518,9 +518,11 @@ def _iterate_trans_views(function, alpha, **kwargs):
 
     try:
         try:
-            return _itv(function, fig, surfaces={"head-dense": alpha}, **kwargs)
+            return _itv(function, fig, **kwargs)
         except OSError:
-            return _itv(function, fig, surfaces={"head": alpha}, **kwargs)
+            # XXX: warn here? ... why do we need this fallback?
+            kwargs["surfaces"] = {"head": alpha}
```

Comment:
For some subjects there might not be a `head-dense` just a sparse head. So this is just a way to prefer the dense one when plotting but fall back to the sparse one if it's not available.